### ISSUE TITLE
Update to Flutter 2

### ISSUE THREAD

**Issue by bhoominn** [OneSignal/OneSignal-Flutter-SDK#359]
**Description:**

(your description here)

**Environment**

**Steps to Reproduce Issue:**

1. (your steps here)
2.
3.

**Anything else:**

(crash stacktraces, as well as any other information here)

**Comment by JobMoll** [comment]
Agreed null safety would be great! This is the only package I have in my app that isn’t null safety ready yet..

**Comment by bhoominn** [comment]
> Agreed null safety would be great! This is the only package I have in my app that isn’t null safety ready yet..

Yes, same here.

**Comment by Jeasmine** [comment]
Thanks for the feedback. This is coming out under major release https://github.com/OneSignal/OneSignal-Flutter-SDK/issues/348
